package/gluon-mesh-batman-adv-core/files/lib/gluon/radvd/arguments

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/lua
+local site = require "gluon.site_config"
+print("-i br-client -p " .. site.prefix6)
+
```

Comment:
*package/gluon-mesh-batman-adv-core/files/lib/gluon/radvd/arguments* is missing the executable flag, this can't work.